core/src/main/scala/cats/data/IorT.scala

```diff
@@ -120,6 +126,31 @@ final case class IorT[F[_], A, B](value: F[Ior[A, B]]) {
 
   def combine(that: IorT[F, A, B])(implicit F: Apply[F], A: Semigroup[A], B: Semigroup[B]): IorT[F, A, B] =
     IorT(F.map2(this.value, that.value)(_ combine _))
+
+  def flatTap[AA >: A, D](f: B => IorT[F, AA, D])(implicit F: Monad[F], AA: Semigroup[AA]): IorT[F, AA, B] =
```

Comment:
Here too.

Reply:
This version allows widening of A. `Monad`'s one doesn't.